examples/graph/mpnn-molecular-graphs.py

```diff
@@ -46,7 +46,7 @@
 [this tutorial](https://keras.io/examples/generative/wgan-graphs/)).
 
 [RDKit](https://www.rdkit.org/) is a collection of cheminformatics and machine-learning
-software written in C++ and Python. In this tutorial, RDKit is used to conviently and
+software written in C++ and Python. In this tutorial, RDKit is used to conveniently and
```

Comment:
Cool, now open the md file and the ipynb file and apply the same change.

Reply:
Sorry @fchollet when I change one line on ipynb by hand it creates additionals 273 changes to that file. I tried jupytext to auto-generate from py file but didn't work.

![Screenshot (511)](https://user-images.githubusercontent.com/10319606/130153954-14002ec7-1dee-41b6-84a3-fb12fe93fa5c.png)

Reply:
Just open the ipynb file in a text editor, modify the one line, and save the file. In fact you could do it directly from the GitHub interface, or in a terminal via `vi`.